servicebox-app/src/main/resources/templates/claims/list.html

```diff
@@ -26,13 +45,15 @@
             </tbody>
```

Comment:
The code:
```
<tbody id="claim-list">
              {#for claim in claims}
                  <!-- Include the HTML template rendering the claim item -->
                  {#include claims/item.html claim=claim /}
              {/for}
            </tbody>
```
Should use the new template `table.html`, so the load page and the search is using the same template.

Reply:
Good idea.

Reply:
See: a6effc0